EpiResponsivePicture.React/epiresponsivepicture-react/example/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "epiresponsivepicture-react-example",
+  "homepage": ".",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ../node_modules/react-scripts/bin/react-scripts.js start",
+    "build": "node ../node_modules/react-scripts/bin/react-scripts.js build",
+    "test": "node ../node_modules/react-scripts/bin/react-scripts.js test",
+    "eject": "node ../node_modules/react-scripts/bin/react-scripts.js eject"
+  },
+  "dependencies": {
+    "@testing-library/jest-dom": "file:../node_modules/@testing-library/jest-dom",
```

Comment:
All types, testing related libs, and typescript should be devDependencies, right?